docs/plugins/transform-react-inline-elements.md

```diff
@@ -10,14 +10,16 @@ Replaces the `React.createElement` function with one that is more optimized for
 
 <blockquote class="babel-callout babel-callout-info">
   <p>
-    Note that this requires the global polyfill due to React's use of ES6 <code>Symbol</code> when validating React Elements.
-    If <code>Symbol</code> is not present in the browser, your application will fail to render, as Babel will polyfill <code>Symbol</code>
-    but React will not have access to that polyfill.
+    Note that this might require the Symbol's global polyfill due to React's use of ES6 <code>Symbol</code> when validating React Elements.
```

Comment:
Good point, we should probably just rewrite the whole section and focus on these edge cases since they'd be most helpful for users.